app/lib/renderer/episode.rb

```diff
@@ -10,7 +10,23 @@ class Episode
     def render
       logger.info "Beginning episode render: #{object.ordinal}: #{object.title}"
       attach_images
+      generate_vtt_text
       render_markdown
     end
+
+    # Extract vtt file
+    def generate_vtt_text
+      return if vtt.blank?
+
+      transcript = " ".dup
+      cue_texts = vtt.cues.collect(&:text).join(' ')
```

Comment:
Sorry to keep going on about this, but I think you can just assign this to the object.transcript directly?

Reply:
Yep! Give that a whirl.

Reply:
Oh, alright!
Just did!